docs/libcurl/curl_unescape.3

```diff
@@ -28,7 +28,7 @@ curl_unescape - URL decodes the given string
 .nf
 #include <curl/curl.h>
 
-char *curl_unescape( const char *url, int length);
+char *curl_unescape(const char *url, int length);
```

Comment:
I think this `url` should be renamed as well to `input`. Primarily to remove the risk that someone would think this operates on an actual URL.